Userland/Shell/Shell.h

```diff
@@ -462,7 +462,8 @@ class Shell : public Core::Object {
 
 #undef __ENUMERATE_SHELL_BUILTIN
 
-            ":"sv, // POSIX-y name for "noop".
+            "."sv, // Alias to "source".
+        ":"sv,     // POSIX-y name for "noop".
```

Comment:
This is the  indentation clang-format gives to me but it looks quite odd, anything wrong here?

Reply:
Yep, looks like a clang-format issue. Please add a `// clang-format off` (and the subsequent `// clang-format on`) comment there (a comment with a brief reason why would be nice too)